### PR TITLE
fix job path, make ENTRYPOINT to work

### PR DESCRIPTION
**Pull request by krzyzacy**
hopefully.. on top of #1404

ref #1348

### CODE REVIEW

File: jobs/ci-kubernetes-node-kubelet-dockerized.sh [hunk 1]
```diff
@@ -22,8 +22,8 @@ set -o xtrace
 
 readonly testinfra="$(dirname "${0}")/.."
 
-export NODE_TEST_SCRIPT="test/e2e_node/jenkins/e2e-node-jenkins.sh"
-export NODE_TEST_PROPERTIES="test/e2e_node/jenkins/jenkins-ci.properties"
+export NODE_TEST_SCRIPT="${REPO_DIR}/test/e2e_node/jenkins/e2e-node-jenkins.sh"
```
Comment (ixdy) [inline jobs/ci-kubernetes-node-kubelet-dockerized.sh]: I think REPO_DIR refers to the path on Jenkins, and won't be valid inside the container.

Comment (krzyzacy) [inline jobs/ci-kubernetes-node-kubelet-dockerized.sh]: https://github.com/kubernetes/test-infra/blob/master/jenkins/node-dockerized.sh#L49

it's mapped but you are right it's pointing to outside where I want. Just made it kubernetes.

**Comment by krzyzacy** [comment]
`docker pull golang:1.7.1` failed on Travis, odd.

**Pull request merged**